src/components/DayPickerRangeController.jsx

```diff
@@ -187,16 +187,21 @@ export default class DayPickerRangeController extends React.Component {
     } = nextProps;
     let { visibleDays } = this.state;
 
+    let recomputeBlockedDays = false;
```

Comment:
Or maybe we even want to separate out this logic such that we're only recalculating the functions that we need to? (e.g. only recompute outside days, blocked days, etc.)